.changeset/empty-coins-yawn.md

```diff
@@ -0,0 +1,9 @@
+---
+'site': minor
+'playground': patch
+'protoform': patch
+'@westpac/test-config': patch
+'@westpac/ts-config': patch
+---
+
+added cms option for design content demo
```

Comment:
We dont need to add a changeset for any website changes just for components so can remove this file